packages/core/admin/admin/src/pages/AuthPage/components/Register/index.js

```diff
@@ -1,173 +1,228 @@
+                    name="password"
+                    onChange={handleChange}
+                    value={values.password}
+                    error={errors.password ? formatMessage({ id: errors.password }) : ''}
+                    endAction={
+                      // eslint-disable-next-line react/jsx-wrap-multilines
+                      <FieldActionWrapper
+                        onClick={e => {
+                          e.preventDefault();
+                          setPasswordShown(prev => !prev);
+                        }}
+                        label={formatMessage({
+                          id: passwordShown
+                            ? 'Auth.form.password.show-password'
+                            : 'Auth.form.password.hide-password',
+                        })}
+                      >
+                        {passwordShown ? <Show /> : <Hide />}
+                      </FieldActionWrapper>
+                    }
+                    required
+                    label={formatMessage({ id: 'Auth.form.password.label' })}
+                    type={passwordShown ? 'text' : 'password'}
+                  />
+                  <TextInput
+                    name="confirmPassword"
+                    onChange={handleChange}
+                    value={values.confirmPassword}
+                    error={
+                      errors.confirmPassword ? formatMessage({ id: errors.confirmPassword }) : ''
+                    }
+                    endAction={
+                      // eslint-disable-next-line react/jsx-wrap-multilines
+                      <FieldActionWrapper
+                        onClick={e => {
+                          e.preventDefault();
+                          setConfirmPasswordShown(prev => !prev);
+                        }}
+                        label={formatMessage({
+                          id: confirmPasswordShown
+                            ? 'Auth.form.password.show-password'
+                            : 'Auth.form.password.hide-password',
+                        })}
+                      >
+                        {confirmPasswordShown ? <Show /> : <Hide />}
+                      </FieldActionWrapper>
+                    }
+                    required
+                    label={formatMessage({ id: 'Auth.form.confirmPassword.label' })}
+                    type={confirmPasswordShown ? 'text' : 'password'}
+                  />
+                  <Checkbox
+                    onValueChange={checked => {
+                      handleChange({ target: { value: checked, name: 'news' } });
+                    }}
+                    value={values.news}
+                    name="news"
+                  >
+                    {formatMessage(
+                      { id: 'Auth.form.register.news.label' },
+                      {
+                        terms: (
+                          <a target="_blank" href="https://strapi.io/terms" rel="noreferrer">
```

Comment:
This color link does not pass the contrast ratio on axe 😬

Reply:
Nice catch !